packages/connect-react/src/components/ConfigureFilePicker/index.tsx

```
@@ -0,0 +1,877 @@
+      </div>
+    );
+  }
+
+  return (
+    <div style={styles.container}>
+      {/* Header with breadcrumbs */}
+      <div style={styles.header}>
+        <div style={styles.breadcrumb}>
+          <span
+            role="button"
+            tabIndex={0}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)